v1/telemetry-worker.js

```diff
@@ -0,0 +1,62 @@
+importScripts("telemetry.js", "big.js")
+
+var gActions = {
+  Histogram_count: function() {
```

Comment:
Just out of curiosity, have you time how much main-thread time is saved from moving these operations to a worker? if not, could you add some console logging code to time these operations and let me know what kinds of numbers you see